community/autossh/APKBUILD

```diff
@@ -27,7 +27,7 @@ prepare() {
 }
```

Comment:
You should also remove `prepare` function; applying of patches is already handled by the `default_prepare` function that is called when no custom `prepare` function is declared.